src/components/Counter/Counter.jsx

```diff
@@ -7,51 +7,59 @@ const colors = {
   red: 'rgb(239, 68, 68)',
 };
 
+const initialState = { count: 0, color: colors.yellow };
+
+function reducer(state, action) {
+  switch (action.type) {
+    case 'increment':
+      return { count: state.count + 1, color: state.color };
+    case 'decrement':
+      return { count: state.count - 1, color: state.color };
+    case 'reset':
+      return { count: 0, color: state.color };
+    case 'positive':
```

Comment:
Here you can also use the spread operator for state since the color will be within state, then designate the count.